napalm/base/__init__.py

```diff
@@ -79,12 +79,17 @@ def get_network_driver(module_name, prepend=True):
         # Try to not raise error when users requests IOS-XR for e.g.
         module_install_name = module_name.replace('-', '')
         # Can also request using napalm_[SOMETHING]
+        community_module_name = "napalm_{name}".format(name=module_install_name)
         if 'napalm' not in module_install_name and prepend is True:
             module_install_name = 'napalm.{name}'.format(name=module_install_name)
+        # Order is custom_napalm_os (local only) ->  napalm.os (core) ->  napalm_os (community)
         try:
             module = importlib.import_module("custom_" + module_install_name)
         except ImportError:
-            module = importlib.import_module(module_install_name)
+            try:
+                module = importlib.import_module(module_install_name)
+            except ImportError:
+                module = importlib.import_module(community_module_name)
```

Comment:
Can we just do a for loop?

custom_install_name = "custom_" + module_install_name
for module_name in [custom_install_name, module_install_name, community_module_name]:
    try:
        ...
        break
    except ImportError:
        pass
    else:
        # Never loaded
        raise ImportError(msg)
```

I think would be clearer to understand than the nested try